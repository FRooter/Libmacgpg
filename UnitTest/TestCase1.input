-----BEGIN PGP MESSAGE-----
Version: GnuPG/MacGPG2 v2.0.17 (Darwin)
Comment: GPGTools - http://gpgtools.org

owGbwMvMwCR4V2ZCVcC/uUaMayKThEJSi0ucE4tTDfVSKwpSk0tSU/x0PJmCU1MV
MkpKCqz09dML0kvy83OK9XIy0zNKMvJLi1MTCwr0kvNz9QuK8rOAWor1zUzNzUz0
SzKTs1OBPENLQ64ONxYGQSYGNlYmkHEMXJwCMFvdzzAs6Au1Of9kyqn/x7gZzyTN
N34569MMdob53moXOA9KR105dWyZaeG36IvL7V8cBgA=
=c8TK
-----END PGP MESSAGE-----
