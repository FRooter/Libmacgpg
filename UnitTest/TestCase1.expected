See http://gpgtools.lighthouseapp.com/projects/65764/tickets/191
